application/models/Template.php

```diff
@@ -48,6 +48,8 @@ class Template extends LSActiveRecord
     /** @var Template - The instance of template object */
     private static $instance;
 
+    public static $sTemplateNameRegex = '/^[a-zA-Z0-9_ -]+$/';
```

Comment:
Not sure this will work in an international context. What if your keyboard is Arabic, Greek or Russian?